README.md

```diff
@@ -52,7 +52,7 @@ As stated above everybody should attempt to complete this task
 
 ## Submission handling
 
-1) Using docker, set up (a) virtual server(s) that can host an `HTCondor` batch sytem. Be inspired by the [HTCondor QuickStart guide](https://htcondor.readthedocs.io/en/latest/admin-manual/quick-start-condor-pool.html).
+1) Using docker, set up (a) virtual server(s) that can host an `HTCondor` batch sytem. Be inspired by the [HTCondor QuickStart guide](https://htcondor.readthedocs.io/en/latest/admin-manual/index.html).
```

Comment:
Yes, the link has changed, but I think it would be better to point directly to https://htcondor.readthedocs.io/en/latest/getting-htcondor/admin-quick-start.html